cmd/ical-relay/main.go

```diff
@@ -60,6 +63,21 @@ func main() {
 
 	log.Tracef("%+v\n", conf)
 
+	if !helpers.DirectoryExists(conf.Server.StoragePath + "notifystore/") {
+		log.Info("Creating notifystore directory")
+		err = os.MkdirAll(conf.Server.StoragePath+"notifystore/", 0750)
+		if err != nil {
+			log.Fatalf("Error creating notifystore: %v", err)
+		}
+	}
+	if !helpers.DirectoryExists(conf.Server.StoragePath + "calstore/") {
+		log.Info("Creating calstore directory")
+		err = os.MkdirAll(conf.Server.StoragePath+"calstore/", 0750)
+		if err != nil {
+			log.Fatalf("Error creating calstore: %v", err)
+		}
+	}
```

Comment:
isnt this in the config parsing part already?

Reply:
I think thats part of the startup process and should therefore not be handled in the config parsing.